app/src/main/java/org/p2p/wallet/renbtc/RenTransactionManager.kt

```diff
@@ -61,12 +61,11 @@ class RenTransactionManager(
         val fee = lockAndMint.estimateTransactionFee()
         Timber.tag(REN_TAG).d("Fee calculated: $fee")
 
-        val session = lockAndMint.session
-        return session
+        return lockAndMint.session
     }
 
-    suspend fun startPolling(session: LockAndMint.Session, secretKey: ByteArray) {
-        if (!this::lockAndMint.isInitialized) throw IllegalStateException("LockAndMint object is not initialized")
+    suspend fun startPolling(session: LockAndMint.Session, secretKey: ByteArray) = withContext(scope.coroutineContext) {
```

Comment:
why not scope.launch?

Reply:
cause it's suspend function , and there is no difference, i just switch context

Reply:
okay i will fix it